Formula/remmina.rb

```diff
@@ -0,0 +1,197 @@
+class Remmina < Formula
+  desc "Remote desktop software client for POSIX-based operating systems"
+  homepage "https://remmina.org/"
+  url "https://gitlab.com/Remmina/Remmina/-/archive/v1.4.29/Remmina-v1.4.29.tar.bz2"
+  sha256 "94d11d00602d12ed033b1018e42171dfd1b1a42081530c229e4a3988c4f3a3d9"
+  license "GPL-2.0-only"
+
+  depends_on "cmake" => :build
+  depends_on "freerdp"
+  depends_on "gtk+3"
+  depends_on "json-glib"
+  depends_on "libvncserver"
+  depends_on "openssl@3"
```

Comment:
you can find the dependency tree issue with `brew deps --tree remmina`

Looks like `freerdp` and `libvncserver` are using `openssl@3` while `unbound`, `libevent` and etc are using `openssl@1.1`

Reply:
How can this be handled? Add `remmina` to `audit_exceptions/versioned_dependencies_conflicts_allowlist.json`?